docs/cpp/source/conf.py

```diff
@@ -26,15 +29,17 @@
 # If your documentation needs a minimal Sphinx version, state it here.
 #
 needs_sphinx = '3.1.2'
+run_doxygen = os.environ.get('RUN_DOXYGEN', "false") == "true"
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
     'sphinx.ext.intersphinx',
+] + ([
     'breathe',
     'exhale'
```

Comment:
Is merely adding these two extensions is all that's needed to run doxygen?

Reply:
https://github.com/pytorch/pytorch/runs/7615170554?check_suite_focus=true seems so!